### PR TITLE
RCT-150: 30 Days of React - Day 6

### PR DESCRIPTION
**Pull request by jhrasco-ct**
## Related Tickets

- [RCT-150: React Training](https://codeandtheory.atlassian.net/browse/RCT-150)

## Description

Completed the exercises in the [30 Days of React - Day 6](https://github.com/jhrasco-ct/30-Days-Of-React/blob/master/06_Day_Map_List_Keys/06_map_list_keys.md).

## Validation

1. Open the JavaScript/HTML file or the React project containing the code for each exercise.
2. Verify that the solutions written are correct and satisfy the challenges.

#### Level 2 Challenges

Screen Shot 2022-11-03 at 7 47 56 PM

Screen Shot 2022-11-03 at 9 46 10 PM

#### Level 3 Challenge

Screen Shot 2022-11-04 at 2 36 20 PM

### CODE REVIEW

File: solutions/day-06/level2/src/components/NumberCell.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const NumberCell = ({ number }) => {
+  let className = null;
+  if (isPrime(number)) {
+    className = "cell-prime";
+  } else if (number % 2 === 0) {
+    className = "cell-even";
+  } else {
+    className = "cell-odd";
+  }
```
Comment (ianabalus) [inline solutions/day-06/level2/src/components/NumberCell.js]: You can instead make the "cell-odd" the default value of the `className` variable, then remove the `else {}` block

File: solutions/day-06/level2/src/components/NumberGrid.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import NumberCell from "./NumberCell";
+
+const NumberGrid = () => {
+  return (
+    <div className="grid">
```
Comment (ianabalus) [inline solutions/day-06/level2/src/components/NumberGrid.js]: Use implicit return if there is no other stuff before the return statement.

**Review by ianabalus — APPROVED** [review]
Approving this PR, but apply the feedback to your future tasks. 👍🏻

**Comment by ianabalus** [comment]
@jhrasco-ct please fix merge conflicts

**Comment by jhrasco-ct** [comment]
@ianabalus, merge conflicts were fixed and feedback was addressed. Thanks.

**Pull request merged**